docs(UnoCheck): Improve Structure to find Troubleshooting and configuration options easyer via Searchengine

DevTKSS opened the pull request:
## What is this PR about

Moved the sub Topics into seperate Pages, so the search engine on the docs homepage can list them and they are more visible in the directory for research purposes of users.
also added the common `There are one or more proplems detected` without uno-check is providing information about it, where we need to run -verbose to see what is its problem
Seperated also the configuration options for same reason, seems to get overseen in the past

[ ]  Breaking Changes => no
[x] Closes issue [#19015](https://github.com/unoplatform/uno/issues/19015) on the main uno repository but has already been closed from creator, its answering additionally recent upcoming discussions to that problem:
[Main Uno repo discussion #14395](https://github.com/unoplatform/uno/discussions/14395)
[Main Uno repo discussion #19079](https://github.com/unoplatform/uno/discussions/19079)

## Anything else?

could we link the Uno check trouble shooting from the main docs trouble shooting area, if the searching person is not first finding this nested area here? Thanks!

DevTKSS commented:
maybe we will need to update the toc.yml file also, please check this in addition, I have no idea how this file has to look like correctly and the docfx guide on contribution did not help because I did not find the discribed Uno.UI-tools.slnf mentioned [here](https://platform.uno/docs/articles/uno-development/docfx.html?tabs=tabid-1#run-docfx-locally) after installation of the docfx via dotnet

jeromelaban commented:
@DevTKSS thanks for your contribution! I've made an additional change to update the TOC. Would you mind rewriting the history for this commit 252bdaffccb64bcece6edcb9edf9ba1589acd97d to be `docs: split doc in multiple pages?`? Thanks!

DevTKSS commented:
@jeromelaban I would do what asked for, but I really dont know what this means... How can I rewrite "the History"? Is that sync fork or something else? Sorry, might sound stupid to a profi...

jeromelaban commented:
See [this doc](https://platform.uno/docs/articles/contributing/guidelines/pull-requests.html#updating-your-branch-on-top-of-the-latest-of-the-default-branch), with the section named `Commit/Pull Request Format`.

DevTKSS commented:
@jeromelaban
I do know that page, but that is not telling (or I might missunderstand it?) how I should rename any commit. currently fighting with my converters branch, where github desktop and vs decided to do as they like and this engines are not seem to care about any format while this.
Intention was originally to pull the online repo down to just delet the left trailing spaces and be fine, but seems like this engine wanted to upload what has been corrected/commited in github web. thats blanc horror at the moment...

jeromelaban commented:
The commands from the docs are exactly what you need to do this, you need to clone the repo, checkout your branch, then execute the commands to change the text of the first commit. You'll find a lot of online documentation on how to make an interactive rebase with the `reword` command.

DevTKSS commented:
@jeromelaban sidenote: since I do see this right now, is it on purpose, the uno.check repo branch of yours is named main instead of master as it is told to cmd in the docs? Wondered why that command failed until I seen that nameing difference of that repo...

and question to your initial message here:

> I've made an additional change to update the TOC. Would you mind rewriting the history for this commit https://github.com/unoplatform/uno.check/commit/252bdaffccb64bcece6edcb9edf9ba1589acd97d to be docs: split doc in multiple pages?? Thanks!

I did now run rebase in vs code and it told it has been rebased and updated, but I the latest commit I can see there is 4959e7c1 from 27 Dec, while on web github I can see the changes in toc.yaml file... If I would now run the git push -f, that would not do what you wanted isnt it? is there some kind of trick after the rebase to the uno-origin/main branch in this case of uno.check?

DevTKSS commented:
did apply changes, but the "rewriting history"... I am reading the interactive rebase on git-scm, which needs to do multiple rebases as it seems to be... should I run the commit with the recent changes applyed first or after this? I dont know it that head rebase to come to that commit mentioned to reword will delete the applyed changes again.

DevTKSS commented:
it just didnt let me do anything before commiting and now the old wrong formated one seems to be gone with commit applyed. would you check if that is now as wanted?

DevTKSS commented:
runned the rename as you told, but to me there is no change visible, sorry if it has not worked... the commit of code changes at least should have been applyed
